internal/beater/server_test.go

```diff
@@ -463,13 +463,14 @@ func TestServerElasticsearchOutput(t *testing.T) {
 	))
 	mp := sdkmetric.NewMeterProvider(sdkmetric.WithReader(reader))
 
+	const maxRequests = 10
 	srv := beatertest.NewServer(t, beatertest.WithMeterProvider(mp), beatertest.WithConfig(agentconfig.MustNewConfigFrom(map[string]interface{}{
 		"output.elasticsearch": map[string]interface{}{
 			"hosts":          []string{elasticsearchServer.URL},
 			"flush_interval": "1ms",
 			"backoff":        map[string]interface{}{"init": "1ms", "max": "1ms"},
 			"max_retries":    0,
-			"max_requests":   10,
+			"max_requests":   maxRequests,
```

Comment:
Will it be easier to just to set it to 1, and assert for 0?
```suggestion
			"max_requests":   1,
```

Reply:
Oh man that is indeed easier... Let me simplify it.